src/DatePickerInput/web/DatePickerInput.js

```diff
@@ -170,13 +171,17 @@ DatePickerInput.propTypes = {
   renderDay: PropTypes.func,
   onDateChange: PropTypes.func,
   theme: PropTypes.object,
+  disabledDays: PropTypes.oneOfType([
+    PropTypes.array, PropTypes.object,
+  ]),
 };
 
 DatePickerInput.defaultProps = {
   placeholder: 'YYYY-MM-DD',
   format: 'YYYY-MM-DD',
   fromMonth: new Date(),
   onDateChange: () => {},
+  disabledDays: [{ before: new Date() }],
```

Comment:
Let's not default this to anything. Since the majority usecase is to allow all dates it at all :)